isisd/isis_sr.c

```diff
@@ -1794,15 +2104,36 @@ int isis_sr_start(struct isis_area *area)
 	struct isis_circuit *circuit;
 	struct listnode *node;
 
+	/* First start Label Manager if not ready */
+	if (!isis_zebra_label_manager_ready())
+		if (isis_zebra_label_manager_connect() < 0) {
+			/* Re-schedule isis_sr_start in 1 sec. to re-attempt
+			 * label manager connection
+			 */
+			thread_add_timer(master, sr_start_label_manager, area,
+					 1, &srdb->t_start_lm);
```

Comment:
this event (`&srdb->t_start_lm`) needs to be canceled when SR is disabled.

Reply:
Done. Add `THREAD_TIMER_OFF(srdb->t_start_lm)` at the beginning of the isis_sr_stop() function